modules/swift/examples/code_snippets/SampleCodeTest.swift

```diff
@@ -1209,2824 +1208,1603 @@ class SampleCodeTest {
+        
     }
-
-
-
-//
-//  URLEndpontListenerTest.swift
-//  CouchbaseLite
-//
-//  Copyright (c) 2020 Couchbase, Inc. All rights reserved.
-//
-//  Licensed under the Couchbase License Agreement (the "License");
-//  you may not use this file except in compliance with the License.
-//  You may obtain a copy of the License at
-//  https://info.couchbase.com/rs/302-GJY-034/images/2017-10-30_License_Agreement.pdf
-//
-//  Unless required by applicable law or agreed to in writing, software
-//  distributed under the License is distributed on an "AS IS" BASIS,
-//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-//  See the License for the specific language governing permissions and
-//  limitations under the License.
-//
-
-// tag::listener[]
-
-// tag::listener-simple[]
-val config =
-  URLEndpointListenerConfiguration(database) // <.>
-
-config.setAuthenticator(
-    ListenerPasswordAuthenticator {
-      username, password ->
-        "valid.user" == username &&
-        ("valid.password.string" == String(password))
+    
+    // MARK: -- Listener
+    
+    func dontTestListenerSimple() throws {
+        let database = try Database(name: "database")
+        // tag::listener-simple[]
+        var config = URLEndpointListenerConfiguration(database: database) // <.>
+        config.authenticator = ListenerPasswordAuthenticator { username, password in
+            return "valid.user" == username && "valid.password.string" == String(password)
+        } // <.>
+        
+        let listener = URLEndpointListener(config: config) // <.>
+        
+        try listener.start()  // <.>
+        
+        // end::listener-simple[]
     }
-) // <.>
-
-val listener =
-  URLEndpointListener(config) // <.>
-
-listener.start()  // <.>
-
-// end::listener-simple[]
-
-
-
-// tag::replicator-simple[]
-
-let tgtUrl = URL(string: "wss://10.1.1.12:8092/actDb")!
-let targetEndpoint = URLEndpoint(url: tgtUrl) //  <.>
-
-var thisConfig = ReplicatorConfiguration(database: actDb!, target: targetEndpoint) // <.>
-
-thisConfig.acceptOnlySelfSignedServerCertificate = true; <.>
-
-let thisAuthenticator = BasicAuthenticator(username: "valid.user", password: "valid.password.string")
-thisConfig.authenticator = thisAuthenticator // <.>
-
-this.replicator = new Replicator(config); // <.>
-
-this.replicator.start(); // <.>
-
-// end::replicator-simple[]
-
-
-
-import XCTest
-@testable import CouchbaseLiteSwift
-
-@available(macOS 10.12, iOS 10.0, *)
-class URLEndpontListenerTest: ReplicatorTest {
-    let wsPort: UInt16 = 4984
-    let wssPort: UInt16 = 4985
-    let serverCertLabel = "CBL-Server-Cert"
-    let clientCertLabel = "CBL-Client-Cert"
-
-    var listener: URLEndpointListener?
-
-    @discardableResult
-    func listen() throws -> URLEndpointListener {
-        return try listen(tls: true, auth: nil)
+    
+    func dontTestListenerInitialize() throws {
+        let otherDB = try Database(name: "otherDB")
```

Comment:
Can we use self.database?

Reply:
we are thinking to use otherDB in case of Listener/OtherSide ? do you think it's not useful?

Reply:
I think we need to differentiate, but agree we need to do so consistently.